src/prognostic_equations/hyperdiffusion.jl

```diff
@@ -133,11 +134,14 @@ function hyperdiffusion_tendency!(Yₜ, Y, p, t)
                 # DSS on Grid scale quantities
                 # Need to split the DSS computation here, because our DSS 
                 # operations do not accept Covariant123Vector types
+                #=
                 @. ᶜ∇²uₕ = C12(ᶜ∇²u)
                 @. ᶜ∇²uᵥ = C3(ᶜ∇²u)
                 dss_op!(ᶜ∇²uₕ, buffer.ᶜ∇²uₕ)
                 dss_op!(ᶜ∇²uᵥ, buffer.ᶜ∇²uᵥ)
                 @. ᶜ∇²u = C123(ᶜ∇²uₕ) + C123(ᶜ∇²uᵥ)
+                =#
+                dss_op!(ᶜ∇²u, buffer.ᶜ∇²u)
```

Comment:
pre-allocate buffer so we can reuse it?

Reply:
I believe this is preallocated in `hyperdiffusion_cache`

Reply:
Oh, i misread it.